python/tvm/target.py

```diff
@@ -311,3 +311,11 @@ def current_target(allow_none=True):
             "Requires a current target in generic function, but it is not set. "
             "Please set it using `with TargetObject:`")
     return Target.current
+
+
+def get_max_num_threads():
+    """Returns the maximum number of threads under current target.
```

Comment:
Can you remove this function and simply use tvm.target.current_target().max_num_threads?

Reply:
Can tvm.target.current_target() be None?

Reply:
I did this because following your usage of max_num_threads.
Like this:
   target = current_target()
   target = target if target else cuda()

Reply:
use  tvm.target.current_target(allow_missing=False)

Reply:
I just think we should remove this function and inline its body to the callers

Reply:
since the function itself is small(one line)

Reply:
ok

tvm.current_target(allow_none=False).max_num_threads
right?

Reply:
k

Reply:
pushed